doc/ref_arch/kubernetes/chapters/chapter01.rst

```diff
@@ -113,25 +113,22 @@ Technology Exceptions
-| Ref           | Name  | Description               | Valid | Rationale                               | Implication |
-|               |       |                           | Until |                                         |             |
-+===============+=======+===========================+=======+=========================================+=============+
-|ra2.exc.tec.001|SR-IOV | This exception allows     | TBD   | Emulation of virtual devices for each   |             |
-|               |       | workloads to use SR-IOV   |       | virtual  machine creates an I/O         |             |
-|               |       | over PCI-PassThrough      |       | bottleneck resulting in poor performance|             |
-|               |       | technology.               |       | and limits the number of virtual        |             |
-|               |       |                           |       | machines a physical server can support. |             |
-|               |       |                           |       | SR-IOV implements virtual devices in    |             |
-|               |       |                           |       | hardware, and by avoiding the use of a  |             |
-|               |       |                           |       | switch, near maximal performance can be |             |
-|               |       |                           |       | achieved. For containerisation the      |             |
-|               |       |                           |       | downsides of creating dependencies on   |             |
-|               |       |                           |       | hardware is reduced as Kubernetes nodes |             |
-|               |       |                           |       | are either physical, or if virtual have |             |
-|               |       |                           |       | no need to "live migrate" as a VNF VM   |             |
-|               |       |                           |       | might.                                  |             |
-+---------------+-------+---------------------------+-------+-----------------------------------------+-------------+
+.. list-table:: Technology Exceptions
+   :widths: 10 10 20 5 50 5
+   :header-rows: 1
+
+   * - Ref
+     - Name
+     - Description
+     - Valid Until
+     - Rationale
+     - Implication
+   * - ra2.exc.tec.001
+     - SR-IOV
+     - This exception allows workloads to use SR-IOV over PCI-PassThrough technology.
+     - TBD
+     - Emulation of virtual devices for each virtual machine creates an I/O bottleneck resulting in poor performance and limits the number of virtual machines a physical server can support. SR-IOV implements virtual devices in hardware, and by avoiding the use of a switch, near maximal performance can be achieved. For containerisation the downsides of creating dependencies on hardware is reduced as Kubernetes nodes are either physical, or if virtual have no need to "live migrate" as a VNF VM might.
```

Comment:
Better to split
```suggestion
     - Emulation of virtual devices for each virtual machine creates an I/O
	   bottleneck resulting in poor performance and limits the number of virtual
	   machines a physical server can support. SR-IOV implements virtual devices
	   in hardware, and by avoiding the use of a switch, near maximal performance
	   can be achieved. For containerisation the downsides of creating
	   dependencies on hardware is reduced as Kubernetes nodes are either physical
	   or if virtual have no need to "live migrate" as a VNF VM might.
```

Reply:
thanks, done in 6c59e87f418ce56e614bb91c7e40ebe3cbab916c